db/schema.rb

```diff
@@ -105,6 +106,16 @@
     t.index ["feature_key", "key", "value"], name: "index_flipper_gates_on_feature_key_and_key_and_value", unique: true
   end
 
+  create_table "itt_providers", force: :cascade do |t|
+    t.text "legal_name"
+    t.text "operating_name"
+    t.datetime "removed_at"
+    t.boolean "approved"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["legal_name"], name: "index_itt_providers_on_legal_name", unique: true
+  end
+
```

Comment:
I don't see a corresponding migration for this change in the PR.

Reply:
oh wow this is a weird commit that came across somewhow!

Reply:
rebasing issue 👍